src/overlays/actors/ovl_En_Karebaba/z_en_karebaba.h

```diff
@@ -7,11 +7,29 @@ struct EnKarebaba;
 
 typedef void (*EnKarebabaActionFunc)(struct EnKarebaba*, GlobalContext*);
 
+enum {
+    /* 0 */ ENKAREBABA_0,
+    /* 1 */ ENKAREBABA_1,
+    /* 2 */ ENKAREBABA_2,
+};
+
 typedef struct EnKarebaba {
     /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x44];
+    /* 0x0144 */ SkelAnime skelAnime;
     /* 0x0188 */ EnKarebabaActionFunc actionFunc;
-    /* 0x018C */ char unk_18C[0x13C];
+    /* 0x018C */ Vec3s jointTable[8];
+    /* 0x01BC */ Vec3s morphTable[8];
+    /* 0x01EC */ u8 unk_1EC;
+    /* 0x01EE */ s16 unk_1EE;
+    /* 0x01F0 */ f32 unk_1F0;
+    /* 0x01F4 */ f32 unk_1F4;
+    /* 0x01F8 */ f32 unk_1F8;
+    /* 0x01FC */ Vec3f unk_1FC;
+    /* 0x0208 */ Vec3f unk_208[2];
+    /* 0x0220 */ Vec3f unk_220;
```

Comment:
```suggestion
    /* 0x01FC */ Vec3f unk_1FC[4];
```
From `func_800BE680`, this is suppose to be `limbPos[4]`